src/link/link_info/ipvlan.rs

```diff
@@ -165,3 +163,24 @@ impl From<IpVlanMode> for u16 {
         }
     }
 }
+
+const IPVLAN_F_PRIVATE: u16 = 0x01;
+const IPVLAN_F_VEPA: u16 = 0x02;
+
+bitflags! {
+    #[non_exhaustive]
+    #[derive(Debug, Clone, Copy, Eq, PartialEq)]
+    pub struct IpVlanFlags: u16 {
+        const PRIVATE = IPVLAN_F_PRIVATE;
```

Comment:
Enum variants should be `UpperCamelCase`.
In this crate, bitflags created type are considered as enum.